src/registrar/tests/test_models_domain.py

```diff
@@ -197,7 +247,10 @@ def test_duplicate_creation(self):
     def tearDown(self) -> None:
         DomainInformation.objects.all().delete()
         DomainApplication.objects.all().delete()
+        PublicContact.objects.all().delete()
         Domain.objects.all().delete()
+        User.objects.all().delete()
+        DraftDomain.objects.all().delete()
```

Comment:
test_approved_application_creates_domain_locally creates a DraftDomain object that doesn't get deleted - would cause issues if these cases expand and need to test on DraftDomain